addons/common/fnc_getGroup.sqf

```diff
@@ -19,6 +19,6 @@ Author:
     Rommel
 
 ---------------------------------------------------------------------------- */
-
-if (toUpper (typeName _this) isEqualTo "GROUP") exitwith {_this};
-group _this
+params ["_group", grpNull];

```

Comment:
You may want to consider
```sqf
params [["_group", grpNull, [grpNull, objNull]]];
```

That should also provide some validation of the input.

Also, `toUpper` may not be necessary. `typeName` returns `"GROUP"` already. The `toUpper` may be redundant here.